app/views_test.py

```diff
@@ -765,7 +765,7 @@ def test_valid_delete_reservation(monkeypatch):
     with app.app_context():
         with app.test_request_context(method='DELETE'):
             def reservation_not_found(_):
-                room = Room(1, False)
+                room = Room(1)
                 user = User('buddy', 'boy')
                 time = Timeslot(1, 2, '2020-01-01', '', 1, 1)
```

Comment:
And here... Now I hope you get an idea why I have been stuck for over 2 hours trying to fix all the tests... LOL